assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -138,10 +150,18 @@ const CartLineItemRow = ( { lineItem } ) => {
 			</td>
 			<td className="wc-block-cart-item__quantity">
 				{ quantitySelector() }
-				<button className="wc-block-cart-item__remove-link">
+				<button
+					disabled={ itemQuantityDisabled }
```

Comment:
Good thinking to disable the quantity selectors while the item removal is pending 👏